viahtml/templates/viahtml/blocked_page.html.jinja2

```diff
@@ -0,0 +1,47 @@
+{% extends "viahtml/wrapper.html.jinja2" %}
```

Comment:
This stuff is all "inspired" by the block pages in Via, but isn't exactly the same. We've merged a few templates into one here.